x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/fleet/agent_details_page/index.tsx

```diff
@@ -97,8 +97,10 @@ export const AgentDetailsPage: React.FunctionComponent = () => {
         <EuiFlexItem>
           <EuiText>
             <h1>
-              {typeof agentData?.item?.local_metadata?.host === 'object' &&
-              typeof agentData?.item?.local_metadata?.host?.hostname === 'string' ? (
+              {isLoading ? (
```

Comment:
we use `pollIntervalMs` on the agent page so you should probably use `isInitialRequest` too to avoid the hostname to blink

Reply:
good catch! thanks, I've updated